cobigen/cobigen-core/src/main/resources/schema/v6.0/templateSetConfiguration.xsd

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema targetNamespace="http://capgemini.com/devonfw/cobigen/TemplateSetConfiguration"
+  xmlns:tns="http://capgemini.com/devonfw/cobigen/TemplateSetConfiguration" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  elementFormDefault="qualified" version="6.0">
+
+  <xs:element name="templateSetConfiguration">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="trigger" type="tns:trigger" minOccurs="0" maxOccurs="unbounded" />
+        <xs:element name="tags" type="tns:tags" minOccurs="1" maxOccurs="1" />
+        <xs:element name="links" type="tns:links" minOccurs="0" maxOccurs="1" />
+        <xs:element name="templates" minOccurs="0" type="tns:templates" />
+        <xs:element name="templateScans" minOccurs="0" type="tns:templateScans" />
+        <xs:element name="increments" minOccurs="0" type="tns:increments" />
```

Comment:
Would it make sense to add a parent element for template relevant tags? (templates, templateScans, increments)
That way you would keep one big advantage of the former seperation: Easy overview of which tags are relevant for context and which for templates

Reply:
Maybe for 6.1? (new issue)